app/components/Sidebar/components/Collections.js

```diff
@@ -20,6 +20,7 @@ type Props = {
 
 function Collections({ onCreateCollection }: Props) {
   const [isFetching, setFetching] = React.useState(false);
+  const [fetchError, setFetchError] = React.useState(false);
```

Comment:
```suggestion
  const [fetchError, setFetchError] = React.useState();
```

`undefined` is better then a boolean in this case